docs/kubernetes-setup.md

```diff
@@ -191,13 +191,19 @@ the string "kube-state-metrics" in it):
       discoveryRule: container_image =~ "kube-state-metrics"
       disableHostDimensions: true
       disableEndpointDimensions: true
+      dimensionTransformations:
+        pod: kubernetes_pod_name
+        namespace: kubernetes_namespace
+        container: container_name
+        node: kubernetes_node
+        namespace_name: destination_namespace
       extraDimensions:
         metric_source: kube-state-metrics
 ```
 
 This uses the [prometheus-exporter](./monitors/prometheus-exporter.md) monitor
 to pull metrics from the service.  It also disables a lot of host and endpoint
-specific dimensions that are irrelevant to cluster-level metrics.  Note that
+specific dimensions that are irrelevant to cluster-level metrics.  `dimensionTransformations` is required to match the dimensions by `kube-state-metrics` to the dimensions generated by other K8s monitors.  Note that
```

Comment:
Won't it be more descriptive if you say, "...`dimensionTransformations` is required to rename the dimensions by ..." instead?